src/improvements/justfile

```diff
@@ -115,6 +115,10 @@ simulate-all-templates:
     root_dir=$(git rev-parse --show-toplevel)
     forge build
     simulation_count=0
+    # When we spawn too many concurrent processes, the simulation fails with 'solc exited with signal: 9 (SIGKILL)'.
+    # The batch size should be low enough to not cause issues, but high enough to not make the simulation too slow.
+    batch_size=13
```

Comment:
Interesting, to know how did you conclude that 13 will be the correct number to avoid the SIGKILL here.